ci_support/run_docker_build.sh

```diff
@@ -37,8 +37,6 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-# library dependencies for matplotlib
-yum install -y libXext libXrender libSM tk libX11-devel
```

Comment:
These, as we already discussed, are covered in the manylinux and will be in the Docker image. However, the issue of specifying some extra yum install is still open.  I am using the `build.sh` as that won't be overwritten by the conda bot when re-generating the feedstock. What do you think?